tests/functional/activitystream/test_activitystream_addon_installed.py

```diff
@@ -8,4 +8,4 @@
 class TestActivityStreamAddonInstalled(object):
 
     def test_activitystream_addon_installed(self, install_xpi, puppeteer):
-        activity_stream = ActivityStream(puppeteer)
+        activity_stream = ActivityStream(puppeteer)  # noqa
```

Comment:
# noqa is prob better than resetting max line length to 100. just my 2 cents since this is pep8 standard.  someone else may prefer 110 or 120  ;)